generated/nitclk/nitclk.proto

```diff
@@ -9,24 +9,17 @@ syntax = "proto3";
 option java_multiple_files = true;
 option java_package = "com.ni.grpc.tclk";
 option java_outer_classname = "NiTClk";
-option csharp_namespace = "NationalInstruments.Grpc.TClk";
+option csharp_namespace = "NationalInstruments.Grpc.Tclk";
 
-package nitclk_grpc;
+package niTClk_grpc;
 
 import "session.proto";
 
 service NiTClk {
   rpc ConfigureForHomogeneousTriggers(ConfigureForHomogeneousTriggersRequest) returns (ConfigureForHomogeneousTriggersResponse);
   rpc FinishSyncPulseSenderSynchronize(FinishSyncPulseSenderSynchronizeRequest) returns (FinishSyncPulseSenderSynchronizeResponse);
-  rpc GetAttributeViReal64(GetAttributeViReal64Request) returns (GetAttributeViReal64Response);
```

Comment:
Guessing all these Get and SetAttribute methods that are missing are marked 'codgen_method': 'no' right now in hapigen. Need to remove those entries so these functions generate again.